weave-scope-cloud-service/README.md

```diff
@@ -0,0 +1,91 @@
+
+While Weave Scope works with Docker and the Weave network, neither is required. Weave Scope can be deployed to any infrastructure, and works well in all cloud and bare-metal environments.
+
+If you haven't already signed up for the cloud service, see [scope.weave.works](http://scope.weave.works) for instructions on getting started. 
+
+You can also run Weave Scope as a stand-alone service, where Weave Scope is run locally to gain insight into your infrastructure. See, ["Using Weave Scope Standalone to Visualize and Monitor Docker Containers"](/guides/weave-scope/weave-scope-alone.html) for more information and an example on how to use it. 
+
+##Running Weave Scope in Cloud Service Mode
+
+Once you've received a service-token id, use it to launch a Weave Scope probe on every machine that you want to monitor:
+
+~~~bash
++sudo wget -O /usr/local/bin/scope \
++  https://github.com/weaveworks/scope/releases/download/latest_release/scope
++sudo chmod a+x /usr/local/bin/scope
+sudo scope launch --service-token=<token>
+~~~
+
+>Note: If you are running Weave Net you can launch a Weave Scope probe onto one host and Weave Net, provided that you have weaveDNS running will discover the rest of the containers. 
+
+##Launching Weave Scope When Using Weave Net to Connect Containers
+
+If you are running Weave Scope on the same machine as the Weave network, the probe uses `weaveDNS` to automatically discover other apps and containers on your network. 
+
+Weave Scope registers itself using the address `scope.weave.local`. Each probe then, sends its report to every app registered under this address. In other words, if you are running Weave Net with DNS, you do not need to take any further steps. Launch once and Weave Net takes care of the rest. 
+
+If you don't want to use weaveDNS, you can instruct Weave Scope to cluster with other Weave Scope instances on the command line. Hostnames and IP addresses are acceptable, both with and without ports, for example:
+
+~~~bash
+scope launch scope1:4030 192.168.0.12 192.168.0.11:4030
+~~~
+
+Hostnames are regularly resolved as A records, where each answer is used as a target.
+
+
+##Visualizing Your Infrastructure
+
+Once Weave Scope is displayed in your browser, it gives you an immediate overview of your network, and shows all of your containerized apps. 
+
+An Overview of the App:![An Overview of the App](/guides/images/weave-scope/weave-scope-application-layers.png)
+
+Each circle on the map represents a container in your network. Circles connected by a line are containers communicating with each other. Use your mouse to explore the network. When you hover over a container or a connection Weave Scope highlights the connected containers to help you understand the topology quicker. This is especially helpful for infrastructures with many containers.
```

Comment:
This also appears to be the first time in this document we use the term topology.  It might be worth describing what it means.  Or not using that term - I try and use view when talking about the UI.